packages/matter-node.js-examples/src/examples/LightDevice.ts

```diff
@@ -50,8 +51,9 @@ const ExampleLight = OnOffLightDevice.with(ReportingOnOffServer);
 // IlluminatedRollerShade.ts for a more detailed example.
 const node = await ServerNode.create();
 
+const part = new Part(ExampleLight);
 // Nodes are a composition of endpoints.  Add a single endpoint to the node, our example light device.
-await node.add(ExampleLight);
+await node.add(part);
```

Comment:
Was adding device type directly not working?

Reply:
Nope ... will revert in next PR